dvc/command/imp.py

```diff
@@ -82,4 +83,15 @@ def add_parser(subparsers, parent_parser):
             "This doesn't affect any DVC operations."
         ),
     )
+    import_parser.add_argument(
+        "-j",
+        "--jobs",
+        type=int,
+        help=(
+            "Number of jobs to run simultaneously. "
```

Comment:
I know this might come from some other DVC commands, but let's reconsider this message please?

`Number of jobs` is not very informative. Number of parallel connections? Number of download jobs?

Reply:
@shcheklein Let's not do that please, it is totally out of scope for this PR.

Reply:
Though I do see your point :slightly_frowning_face:

Reply:
Could just refer to `dvc pull/fetch/status` here, to smooth this out. E.g. `Please refer to `dvc fetch` help for description` or something like that.

Reply:
I'm not referring to push/pull or any other commands (we can create a ticket for this if needed). `this message` in my comment was about this specific help message only.

Reply:
@shcheklein I understand that. This one was copied over from push/pull/etc, so the questions might arise there as well. If we use `Please refer to dvc fetch ...` here we'll dodge the bullet :smile:

Reply:
hmm ... I guess, my take on this it's fine to change only here (and later propagate if needed, and if it's needed at all). Also `Please refer to dvc fetch` will force us to kinda go and see `fetch`, and what's there, etc, etc. Also it complicates UX. To be honest, I haven't see that kind of redirects in the help messages.

Reply:
I agreed that we can change it here, and submit another one for push/pull/etc.

Reply:
In fact per #4838 this is `--jobs` option has a very specific mening? "This external tracked data might be stored in a remote DVC repository. In this situation --job which controls the parallelism level for DVC to download data from remote storage." so seems like it's not really the same as in other commands?

> Please refer to dvc fetch will force us to kinda go and see fetch... I haven't see that kind of redirects in the help messages.

Unrelated, but `dvc get-url -h` used to refer to `import-url` (for the `url` arg. details) I think. Now only `import-url` has the full list of URLs.

Reply:
@jorgeorpinel `dvc import` clones an external repo, and then pull down the data. So I think it means the same as what in `dvc pull`.